content/logs/processing/parsing.md

```diff
@@ -99,7 +99,7 @@ Here is the list of all the matchers and filters natively implemented by Datadog
 | `json`                                                         | parses properly formatted JSON                                                                                                                      |
 | `rubyhash`                                                     | parses properly formatted Ruby Hash (eg {name => "John" "job" => {"company" => "Big Company", "title" => "CTO"}})                                   |
 | `geoip`                                                        | parses an IP or a host and returns a JSON object that contains the continent, country, city and location of the IP address.                         |
-| `useragent([decodeuricomponent:true/false])`                   | parses a user-agent and returns a JSON object that contains the device, os and the browser represented by the Agent. [More info](#useragent-parser) |
+| `useragent([decodeuricomponent:true/false])`                   | parses a user-agent and returns a JSON object that contains the device, os and the browser represented by the Agent. [More info][3] |
```

Comment:
we should ideally avoid "click here" and "more info" links

http://stephanieleary.com/2015/05/why-click-here-is-a-terrible-link-and-what-to-write-instead/